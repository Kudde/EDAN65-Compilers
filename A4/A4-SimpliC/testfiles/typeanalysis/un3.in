int fun() {
  int a = 3;
  a = (5 >= 1) + 3;
  if (2 * 1) {
    return 2;
  }
}
