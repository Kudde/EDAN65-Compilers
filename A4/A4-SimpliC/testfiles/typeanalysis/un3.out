Error at line 3: type boolType should be intType
