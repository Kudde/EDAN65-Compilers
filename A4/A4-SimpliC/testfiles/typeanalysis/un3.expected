Error at line 4: Incompatible types!
